scripts/notify-new-listings.js

```diff
@@ -0,0 +1,126 @@
+// This example uses events to detect when new listings are pending approval or
+// are published and prints out information about those listings. The sequence
+// ID of the last processed event is stored locally so that the event processing
+// can continue from the correct point on next execution.
+
+// This dotenv import is required for the `.env` file to be read
+require('dotenv').config();
+fs = require('fs');
+
+const flexIntegrationSdk = require('sharetribe-flex-integration-sdk');
+
+const integrationSdk = flexIntegrationSdk.createInstance({
+  // These two env vars need to be set in the `.env` file.
+  clientId: process.env.FLEX_INTEGRATION_CLIENT_ID,
+  clientSecret: process.env.FLEX_INTEGRATION_CLIENT_SECRET,
+
+  // Normally you can just skip setting the base URL and just use the
+  // default that the `createInstance` uses. We explicitly set it here
+  // for local testing and development.
+  baseUrl: process.env.FLEX_INTEGRATION_BASE_URL || 'https://flex-integ-api.sharetribe.com',
+});
+
+// Start polloing from current time on, when there's no stored state
+const startTime = new Date();
+
+// Polling interval (in ms) when all events have been fetched.
+// Keeping this at 10s or more is a good idea.
+const pollIdleWait = 10000;
+// Polling interval (in ms) when a full page of events is received and there may be more
+const pollWait = 250;
+
+// File to keep state across restarts. Stores the last seen event sequence ID,
+// which allows continuing polling from the correct place
+const stateFile = "./notify-new-listings.state";
+
+const queryEvents = (args) => {
+  var filter = {eventTypes: "listing/created,listing/updated"};
+  return integrationSdk.events.query(
+    {...args, ...filter}
+  );
+};
+
+const saveLastEventSequenceId = (sequenceId) => {
+  try {
+    fs.writeFileSync(stateFile, sequenceId);
+  } catch (err) {
+    throw err;
+  }
+};
+
+const loadLastEventSequenceId = () => {
+  try {
+    const data = fs.readFileSync(stateFile);
+    return parseInt(data, 10);
+  } catch (err) {
+    return undefined;
```

Comment:
```suggestion
    return null;
```
- `null` is something that is used for the intentional absence of any object value. 
- `undefined` would work too, because you are assigning a variable (called "undefined"), which is undefined in practice. However, since you are handling the situation with "try - catch", I'd set that the null is a better value to be returned.

The use case for "undefined" is something like this:

```js
const obj = {};
const a = obj.a;
if (typeof a === 'undefined') {
  console.log('variable "a" was undefined.');
}
```